docs/sources/tutorials/grafana-fundamentals/index.md

```diff
@@ -27,6 +27,13 @@ In this tutorial, you'll learn how to use Grafana to set up a monitoring solutio
 - Annotate dashboards
 - Set up alerts
 
+Alternatively, you can also watch our Grafana for Beginners series where we discuss fundamental concepts to help you get started with Grafana.
+
+<div style="position:relative;padding-bottom:56.25%;height:0;overflow:hidden">
+   <iframe src="https://www.youtube.com/embed/videoseries?si=ueLa_QEXz20IWnGt&amp;list=PLDGkOdUX1Ujo27m6qiTPPCpFHVfyKq9jT" style="position:absolute;top:0;left:0;width:100%;height:100%;border:0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
+   </iframe>
+</div>
```

Comment:
We don't currently have a short code for displaying a YouTube playlist. So, I've added the iframe code instead.